v1.0-rc.2.md

```diff
@@ -0,0 +1,88 @@
+- The [`cockroach sql`](use-the-built-in-sql-client.html) shell no longer buffers all results in memory before printing them. [#15277](https://github.com/cockroachdb/cockroach/pull/15277)
+- The log file is now flushed more reliably when a node exits. [#15531](https://github.com/cockroachdb/cockroach/pull/15531)
+- The new flag `--max-offset` on the [`cockroach start`](start-a-node.html) command defines the maximum allowed clock offset for the cluster and replaces the environment variable `COCKROACH_MAX_OFFSET`. Note that the `--max-offset` value must be the same on all nodes in the cluster. In order to change it, every node in the cluster must be stopped and restarted with the new value. [#15568](https://github.com/cockroachdb/cockroach/pull/15568)
+- The `--raft-tick-interval` flag has been removed from the [`cockroach start`](start-a-node.html) command. [#15547](https://github.com/cockroachdb/cockroach/pull/15547)
+- Surface errors encountered during certificate and key load. [#15655](https://github.com/cockroachdb/cockroach/pull/15655)
+
+### Admin UI Changes
+
+- The number of transactions that would be affected by a change in isolation level is now tracked. [#15234](https://github.com/cockroachdb/cockroach/pull/15234)
+- Improved style of the single node and log pages. [#15486](https://github.com/cockroachdb/cockroach/pull/15486)
+- The event log now refreshes automatically. [#15502](https://github.com/cockroachdb/cockroach/pull/15502)
+- Add tooltips for graphs on overview page. [#15624](https://github.com/cockroachdb/cockroach/pull/15624)
+
+### Bug Fixes
+
+- Fixed a panic when a query returning a constant `NULL` uses the new distributed execution engine. [#15530](https://github.com/cockroachdb/cockroach/pull/15530)
+- Fixed two issues that could cause requests to get stuck indefinitely. [#15557](https://github.com/cockroachdb/cockroach/pull/15557) [#15573](https://github.com/cockroachdb/cockroach/pull/15573)
+- Fixed a rare crash when extending table leases. [#15560](https://github.com/cockroachdb/cockroach/pull/15560)
+- Fixed a crash when a transaction that uses [`SAVEPOINT cockroach_restart`](savepoint.html) does something other than `ROLLBACK TO SAVEPOINT` after an error. [#15563](https://github.com/cockroachdb/cockroach/pull/15563)
+- Fixed [`RESTORE`](restore.html) for interleaved tables. [#15564](https://github.com/cockroachdb/cockroach/pull/15564)
+- Return parse error instead of panic when constant typechecking fails. [#15636](https://github.com/cockroachdb/cockroach/pull/15636)
+- Prevent persistent distsql and schema change errors after a node dies. [#15368](https://github.com/cockroachdb/cockroach/pull/15368)
+- Adding a new column via [`ALTER TABLE ADD COLUMN`](add-column.html) with a `DEFAULT unique_rowid()` specification no longer generates duplicate values. [#15628](https://github.com/cockroachdb/cockroach/pull/15628)
```

Comment:
"no longer generates" -> "much more rarely generates"
(they still may generated if a node's clock jumps backwards during the back fill)